src/types/admin.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod";
+
+export const EditAdminValidator = z.object({
+    firstName : z.string().trim().min(3).regex(/^[a-zA-Z0-9_ ]+$/),
+    lastName : z.string().trim().min(3).regex(/^[a-zA-Z0-9_ ]+$/),
+    email : z.string().email().min(5),
+    department : z.string(),
+    role: z.enum(['Student', 'Faculty']).optional(),
```

Comment:
- Role and isActive need to be checked and should not be a part of the validator.
- Department should be trimmed and must have a minimum of 3 characters.

Reply:
Removed

Reply:
added z.string().min(3) for department